src/explore-education-statistics-admin/src/pages/methodology/components/MethodologySummaryForm.tsx

```diff
@@ -25,81 +25,77 @@ const errorMappings = [
 ];
 
 interface Props {
-  id: string;
-  initialValues?: MethodologySummaryFormValues;
   defaultTitle: string;
+  id: string;
+  initialValues: FormValues;
   submitText: string;
   onCancel: () => void;
-  onSubmit: (values: MethodologySummaryFormValues) => void;
+  onSubmit: (title: string) => Promise<void>;
```

Comment:
Could make this a little more flexible by using `Promise<void> | void`. This way it doesn't need to be async callback and `noop` would work in the tests

Reply:
Cool, I've changed it now.